src/app/features/fss-search/fss-simplified-search/fss-simplified-search.component.scss

```diff
@@ -28,4 +28,27 @@
         flex-grow: 1;
 		margin-left: 70px;
     }
-  }
\ No newline at end of file
+}
+.switchToESS{
+	width: 100%; 
+	float: right;
+    text-align: right;
+	padding: 0px;
+}
+	.Exchangesetpara{
```

Comment:
Please follow the existing naming convention for all the new CSS classes (camel case)

For example: `.Exchangesetpara` should be `.exchangeSetPara`

Reply:
resolved by adding changes